tests/client_secrets.json

```diff
@@ -1,12 +1,9 @@
 {
-    "web": {
-        "auth_uri": "https://test/auth",
-        "client_id": "MyClient",
-        "client_secret": "MySecret",
-        "redirect_uris": [
-            "http://localhost:5000/oauth2callback"
-        ],
-        "token_uri": "https://test/token",
-        "token_introspection_uri": "https://test/tokeninfo"
-    }
+  "OIDC_CLIENT_ID": "MyClient",
```

Comment:
This should not be changed, as this project does not control the format of `client_secrets.json`.